packages/instant/src/redux/analytics_middleware.ts

```diff
@@ -53,6 +53,30 @@ export const analyticsMiddleware: Middleware = store => next => middlewareAction
                 ).toString();
                 analytics.addUserProperties({ ethBalanceInUnitAmount });
             }
+            break;
+        case ActionTypes.UPDATE_SELECTED_ASSET:
+            const selectedAsset = curState.selectedAsset;
+            if (selectedAsset) {
+                const assetName = selectedAsset.metaData.name;
```

Comment:
I think asset data only is ok? right now by coincidence we have the `name` property available in both `ERC20AssetMetaData` and `ERC721AssetMetaData` but I can see future asset types not necessarily having a "name". Assetdata should be good enough for us to differentiate the assets.

Reply:
I think being able to easily view the name in heap instead of having to reference an asset data mapping would be a huge benefit